crates/krun-guest/src/cli_options.rs

```diff
@@ -10,7 +10,7 @@ pub struct Options {
     pub username: String,
     pub uid: Uid,
     pub gid: Gid,
-    pub command: String,
+    pub command: Option<String>,
```

Comment:
I never realized this when I did the rewrite, but shouldn't we use `PathBuf` for command?

Reply:
Yes, let's do that in a follow-up PR.

Reply:
#28